notifier/main.go

```diff
@@ -57,25 +44,35 @@ func main() {
 		fmt.Printf("Moira notifier version: %s\n", Version)
 		os.Exit(0)
 	}
-	if err := readSettings(*configFileName); err != nil {
+	var err error
+	if config, err = readSettings(*configFileName); err != nil {
 		fmt.Printf("Can not read settings: %s \n", err.Error())
 		os.Exit(1)
 	}
-
+	notifier.SetSettings(config)
 	if err := configureLog(); err != nil {
 		fmt.Printf("Can not configure log: %s \n", err.Error())
 		os.Exit(1)
 	}
-	notifier.InitRedisDatabase()
 	if err := configureSenders(); err != nil {
 		log.Fatalf("Can not configure senders: %s", err.Error())
 	}
+	if err := notifier.CheckSelfStateMonitorSettings(); err != nil {
+		log.Fatalf("Can't configure self state monitor: %s", err.Error())
+	}
+	notifier.InitRedisDatabase()
 	notifier.InitMetrics()
 
 	shutdown := make(chan bool)
 	var wg sync.WaitGroup
 	run(notifier.FetchEvents, shutdown, &wg)
 	run(notifier.FetchScheduledNotifications, shutdown, &wg)
+	if config.Notifier.SelfState.Enabled == "true" {
```

Comment:
Case sensitive comparison. Better to convert config.Notifier.SelfState.Enabled to boolean